src/classes/ADDR_Validator_TDTM.cls

```diff
@@ -58,7 +60,8 @@ public class ADDR_Validator_TDTM extends TDTM_Runnable {
         
         if(!alreadyRun) {
 	        Addr_Verification_Settings__c settings = UTIL_CustomSettingsFacade.getAddressVerificationSettings();
-	        if(trigger.isBefore && settings.enable_automatic_verification__c) {        	
+	        if ((triggerAction == TDTM_Runnable.Action.BeforeInsert || triggerAction == TDTM_Runnable.Action.BeforeUpdate) &&
```

Comment:
So the Trigger context variable only exists in the trigger itself, not in a class called from it? I just want to understand it to make sure I don't make the wrong assumption in the future...